tox.ini

```diff
@@ -27,7 +27,7 @@ setenv=
     SIDEBOARD_CONFIG_OVERRIDES=test-defaults.ini
 deps= -rrequirements.txt
 commands=
-    coverage run --source sideboard -m py.test {posargs}
+    coverage run --source sideboard -m py.test {posargs} sideboard
```

Comment:
This was preventing tests from running in our Vagrant deployments, because it would try to test everything in the plugins directory as well.